source/includes/steps-fle-configure-the-mongodb-client.yaml

```diff
@@ -181,4 +226,17 @@ content: |
                  extraOptions: extraOptions,
                }
            });
+     .. tab::
+        :tabid: python
+
+        .. code-block:: python
```

Comment:
Could you make sure to emphasize important lines similar to the other driver examples?